docs/book/how-to/model-management-metrics/track-metrics-metadata/attach-metadata-to-a-model.md

```diff
@@ -1,62 +1,90 @@
 ---
-description: >-
-  Attach any metadata as key-value pairs to your models for future reference and
-  auditability.
+description: Learn how to attach metadata to a model.
 ---
 
 # Attach metadata to a model
```

Comment:
This sections deserves an addition to explain people how they shall handle models from context (https://github.com/zenml-io/template-llm-finetuning/pull/11#discussion_r1843281476)